niworkflows/reports/core.py

```diff
@@ -463,7 +463,8 @@ def _process_orderings(orderings, layout):
         ]
         # if all values are None for an entity, we do not want to keep that entity
         keep_idx = [
-            not (len(val_set) == 1 and None in val_set or not val_set) for val_set in unique_values
+            not ((len(val_set) == 1 and None in val_set) or not val_set)
+            for val_set in unique_values
```

Comment:
This seems like a more readable check:

```suggestion
            val_set not in (set(), {None}) for val_set in unique_values
```